### PR TITLE
Allow users to create function without dependencies

### PR DESCRIPTION
**Pull request by pPrecel**
**Description**

Changes proposed in this pull request:

- as in the title

**Related issue(s)**

Fixes https://github.com/kyma-project/kyma/issues/15339

### CODE REVIEW

**Review by kwiatekus — APPROVED** [review]
tested on local busola instance.
Works as expected.
Empty dependencies are allowed ( defaulted to `{}` by serverless backend )

File: core-ui/src/components/Functions/FunctionDetails/Tabs/Code/CodeAndDependencies/CodeAndDependencies.js [hunk 1]
```diff
@@ -72,6 +72,7 @@ export default function CodeAndDependencies({ func }) {
     }
 
     setDisabledCause(DISABLED_CAUSES.VALID);
+    setDependencies(deps);
```
Comment (Wawrzyn321) [inline core-ui/src/components/Functions/FunctionDetails/Tabs/Code/CodeAndDependencies/CodeAndDependencies.js:75]: When is it needed? I removed it and it works just fine.

Comment (pPrecel) [reply core-ui/src/components/Functions/FunctionDetails/Tabs/Code/CodeAndDependencies/CodeAndDependencies.js:75]: It's needed when dependencies input from UI contains only spaces, `\n` or `\t`. If you would pass only these symbols in the dependencies on the UI, then the function would report a build error.

**Review by Wawrzyn321 — APPROVED** [review]

**Pull request merged**